source/pmc/community-growth.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Community Growth
+tags: ["pmc","committers","community"]
+---
+
+Growing your project community takes work. This document makes practical
+suggestions of how to encourage people to get involved in your project.
+
+## Publish a road map
+
+Volunteers can work on whatever they want, and so we have a tendency to
+not want to tell them what to work on. However, publishing a roadmap, or
+even a wishlist of features or improvements that are desired, can give
+people an idea of where they might be able to get involved.
+
+This also gives a clear sense that the project has ambitions, and is
+going somewhere, rather than that it is stagnating.
+
+## Periodically publish the open issue list
+
+Your ticket tracker allows you to periodically summarize the open
+issues. Sending this to the dev@ list reminds people that there are
+things to work on, and gives them permission, and encouragement, to do
+so.
+
+## Clearly document contribution processes
+
+Double-check that your "how to contribute / build / test / submit PR"
+documentation is super clear and easy to follow.  Long-time committers
+on a project often forget all the institutional knowledge they just
+"know", so ensuring the "getting started" document actually works
+for newcomers is always worth looking at.
+
+Encourage newcomers to talk about, and document, their pain points
```

Comment:
I would also mention the proposal process: how to submit new ideas ? how to be sure no proposal is lost or stale for too long ? how to track the proposals under discussion ?